api/src/main/java/org/openmrs/Provider.java

```diff
@@ -92,6 +96,48 @@ public String getIdentifier() {
 		return identifier;
 	}
 	
+	/**
+	 * Sets the role concept
+	 * 
+	 * @since 2.2
+	 * @param role the role to set
+	 *                
+	 */
+	public void setRole(Concept role){
+		this.role = role;
+	}
+
+	/**
+	 * Gets the role concept
+	 * 
+	 * @since 2.2
+	 * @return the role
+	 * 
```

Comment:
To be consistent, can we remove this line?

Reply:
Same here, this is the right thing to do